src/components/CheckList.tsx

```diff
@@ -648,6 +661,12 @@ export const CheckList = ({ instance, onAddNewClick, checks, onCheckUpdate }: Pr
           )}
         </div>
       )}
+      <ThresholdGlobalSettings
+        onDismiss={() => setShowThresholdModal(false)}
```

Comment:
question: does onDismiss get called onSuccess? Or I guess the more relevant question is do we want the modal to close on successful select? I could go either way, just thinking aloud

Reply:
Yep, upon successful update, `onSuccess` and `onDismiss` are called in succession.

https://github.com/grafana/synthetic-monitoring-app/pull/338/files#diff-e435217755667714e83a32a6ccbf2072b56705993b58c69d44fa7b78cef70efaR58